### DISCUSSION TITLE
Background element outputting by default now?

### DISCUSSION THREAD

**Discussion by gjblajian** [dolanmiu/docx#1821] [Q&A] [answered]
@dolanmiu 
"docx": "7.6.0"

My input contains no references to background yet the output is consistently placing an empty background element in the xml. Is this new? Did I just not notice it before? Why are we adding elements that are not present in typical native word/OO xml?

![image](https://user-images.githubusercontent.com/5183939/205959138-38cc7d10-edeb-44d3-8a4a-a0797be345e4.png)

**Comment by dolanmiu** [comment, marked as answer]
It is new yes

I thought that adding empty tags do no harm and simplifies code logic, so why not?

Do you notice a difference in your generated documents?

https://github.com/dolanmiu/docx/blob/master/src/file/document/document.ts#L76

**Reply by gjblajian** [reply]
I did not notice that the presence of the empty element caused an issue in MS Word but I am not sure it did not cause an issue when attempting to parse (import) that document using another library. I am not saying that it did but now there are additional factors at play that  have to be investigated.

**Reply by gjblajian** [reply]
would it hurt to wrap the root.push in an if so that it is only pushed when the background property is present?

**Reply by dolanmiu** [reply]
> would it hurt to wrap the root.push in an if so that it is only pushed when the background property is present?

That is possible yes

I will do that

**Reply by dolanmiu** [reply]
Cool, done, this PR solves the issue:

https://github.com/dolanmiu/docx/pull/1830

**Reply by gjblajian** [reply]
@dolanmiu Thank you.
